CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/InfoMapPanel.java

```diff
@@ -42,6 +42,8 @@ public Config getConfig() {
             config.setConnectionType(Config.ConnectionType.EDGES);
         } else if (rbLinks.isSelected()) {
             config.setConnectionType(Config.ConnectionType.LINKS);
+        } else {
+            // Default case added per S126
```

Comment:
I would change this comment to "Do nothing" as this comment doesn't make much sense outside the sonar context. Similarly for other comments like this one